lib/internal/errors.js

```diff
@@ -404,7 +405,7 @@ function uvErrmapGet(name) {
   if (!uvBinding.errmap) {
     uvBinding.errmap = uvBinding.getErrorMap();
   }
-  return uvBinding.errmap.get(name);
+  return MapPrototypeGet(uvBinding.errmap, name);
```

Comment:
Isn't using `SafeMap` should be the way to go? Might be missing some context.

Reply:
It should indeed, but this map is created in C++ which hasn't access to `SafeMap`:

https://github.com/nodejs/node/blob/c380ee6785b72fd2f24803de80bb412dfc32e8a9/src/uv.cc#L79-L101